staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -492,7 +492,8 @@ func installAPI(s *GenericAPIServer, c *Config) {
 	routes.Version{Version: c.Version}.Install(s.Handler.GoRestfulContainer)
 
 	if c.EnableDiscovery {
-		s.Handler.GoRestfulContainer.Add(s.DiscoveryGroupManager.WebService())
```

Comment:
This was capturing traffic for `/apis/apiextensions.k8s.io/v1alpha1` and underneath.  We don't want that behavior from the gorestful routes.